.github/workflows/cri-dwp-kbv-stub.yml

```diff
@@ -0,0 +1,101 @@
+name: DWP KBV CRI Stub - Secure Pipeline build, push & Ship
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - di-ipv-credential-issuer-stub/deploy/dwp-kbv/*
+      - di-ipv-credential-issuer-stub/gradle/**
+      - di-ipv-credential-issuer-stub/src/**
+      - di-ipv-credential-issuer-stub/*
+      - .github/workflows/cri-dwp-kbv-stub.yml
+  workflow_dispatch:
+
+jobs:
+  dockerBuildAndPush:
+    name: Docker build and push
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    env:
+      AWS_REGION: eu-west-2
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+
+      - name: Set up AWS creds
+        uses: aws-actions/configure-aws-credentials@v4.0.2
+        with:
+          role-to-assume: ${{ secrets.CRI_DWP_KBV_GH_ACTIONS_ROLE_ARN }}
```

Comment:
I guess we need to add these github secrets?

Reply:
Correct, me and Vam will run the pipelines manually in stubs-common-infra to get the outputs. And he will add the secrets to this repo

Reply:
These have now been added